addons/account_edi/models/account_edi_document.py

```diff
@@ -18,6 +18,7 @@ class AccountEdiDocument(models.Model):
     attachment_id = fields.Many2one('ir.attachment', help='The file generated by edi_format_id when the invoice is posted (and this document is processed).')
     state = fields.Selection([('to_send', 'To Send'), ('sent', 'Sent'), ('to_cancel', 'To Cancel'), ('cancelled', 'Cancelled')])
     error = fields.Html()
+    error_level = fields.Selection(selection=[('info', 'Info'), ('warning', 'Warning'), ('error', 'Error')])
```

Comment:
```
    message = fields.Html()
    message_level = fields.Selection(selection=[('info', 'Info'), ('warning', 'Warning'), ('error', 'Error')])
``` 
?

Reply:
I would prefer to keep 'error' as these are shown in a special tree view via the banner. 'Message' would confuse the developer making them think that they should return a message when like 'EDI successfully submitted to government', and the banner would show (polluting the view) whereas they should instead post it themselves on the chatter to keep track.